src/cli/commands/config/replace.js

```diff
@@ -3,7 +3,7 @@ const debug = require('debug')
 const path = require('path')
 const log = debug('cli:config')
 log.error = debug('cli:config:error')
-const utils = require('utils')
+const utils = require('../../utils')
```

Comment:
@diasdavid sure thing, will look at it later today also, so maybe I can help you out @xicombd.